int/api/myplugin/execute.go

```diff
@@ -0,0 +1,77 @@
+package myplugin
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/thyra/api/swagger/server/models"
+	"github.com/massalabs/thyra/api/swagger/server/restapi/operations"
+	"github.com/massalabs/thyra/pkg/plugin"
+)
+
+func newExecute(manager *plugin.Manager) operations.PluginManagerExecuteCommandHandler {
+	return &execute{manager: manager}
+}
+
+type execute struct {
+	manager *plugin.Manager
+}
+
+//nolint:cyclop
+func (c *execute) Handle(params operations.PluginManagerExecuteCommandParams) middleware.Responder {
+	plugin, err := c.manager.Plugin(params.ID)
+	if err != nil {
+		return operations.NewPluginManagerExecuteCommandNotFound().WithPayload(
+			&models.Error{Code: errorCodePluginUnknown, Message: fmt.Sprintf("get plugin error: %s", err.Error())})
+	}
+
+	status := plugin.Status()
+
+	pluginName := filepath.Base(plugin.BinPath)
+
+	cmd := params.Body.Command
+	switch cmd {
+	case "start":
+		err := plugin.Start()
+		if err != nil {
+			return executeFailed(cmd, status,
+				fmt.Sprintf("Error while starting plugin %s: %s.\n", pluginName, err))
+		}
+	case "stop":
+		err := plugin.Stop()
+		if err != nil {
+			return executeFailed(cmd, status, fmt.Sprintf("Error while stopping plugin %s: %s.\n", pluginName, err))
+		}
+	case "restart":
+		err := plugin.Stop()
+		if err != nil {
+			return executeFailed(cmd, status, fmt.Sprintf("Error while stopping plugin %s: %s.\n", pluginName, err))
+		}
+
+		err = plugin.Start()
+		if err != nil {
+			return executeFailed(cmd, status,
+				fmt.Sprintf("Error while restarting plugin %s: %s.\n", pluginName, err))
+		}
+	case "update":
```

Comment:
Not implemented ?

Reply:
No.... I don't have the spec 🥲 
more seriously, what this should do?

Reply:
My suggestion was to return a [501 - Not implemented](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501) :rofl:.

Wdyt?